.jenkins/perf_test

```diff
@@ -0,0 +1,731 @@
+//def shared_library_branch = scm.branches[0].name
+//if (shared_library_branch .contains("*/")) {
+//    shared_library_branch  = shared_library_branch.split("\\*/")[1]
+//   }
+//def util_lib="jenkins-shared@${shared_library_branch}"
+//echo "${util_lib}"
+
+//library "${util_lib}"
+
+pipeline {
+  agent { node { label 'austin' } }
+    environment {
+      docker_args="--device=/dev/kfd --device=/dev/dri --group-add video --group-add render --cap-add=SYS_PTRACE --security-opt seccomp=unconfined -v=/var/jenkins/:/var/jenkins"
+      extra_log_env   = " MIOPEN_LOG_LEVEL=5 "
+      Fp16_flags      = " -DMIOPEN_TEST_HALF=Off"
+      Bf16_flags      = " -DMIOPEN_TEST_BFLOAT16=Off"
+      Int8_flags      = " -DMIOPEN_TEST_INT8=Off"
+      Full_test       = " -DMIOPEN_TEST_ALL=Off"
+      Smoke_targets   = " check MIOpenDriver"
+      NOCOMGR_flags   = " -DMIOPEN_USE_COMGR=Off"
+      NOMLIR_flags    = " -DMIOPEN_USE_MLIR=Off"
+  }
+  parameters {
+    string(name: 'PERF_TEST_OVERRIDE', defaultValue: '',  description: 'Add extra env vars for the MIOpenDriver cmd, comma separated')
+    booleanParam(name: 'PERF_TEST_FP32', defaultValue : true, description: 'Run FP16 tests')
+    booleanParam(name: 'PERF_TEST_FP16', defaultValue : true, description: 'Run FP16 tests')
+    booleanParam(name: 'PERF_TEST_ARCHIVE', defaultValue : true, description: 'Archive results from this run')
+  }
+  stages {
+        stage('Build Docker'){
+            steps{
+                script {
+                 echo "Build Docker Stage"
+                def utils = load "vars/utils.groovy"
+                utils.getDockerImage()
+                }
+            }
+        }
+        /*stage("Packages") {
+            parallel {
+                stage("HIP Package") {
+                    agent{ node { label 'nogpu'} }
+                    steps{
+                        script {
+                        echo "Packages Stage"
+                        def utils = load "vars/utils.groovy"
```

Comment:
@cderb can I do this somewhere else, do i dont have to call load before any call to util.something?

Reply:
You should be able to include a reference to the utils.groovy at the head of the file:
https://github.com/ROCm/MITuna/blob/8beff11d8a8b890dc8080591192f4732d3d84d99/.jenkins/perf_eval#L1-L8